<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>input_dialog</class>
 <widget class="QDialog" name="input_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Input</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLineEdit" name="txt_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>33</pointsize>
    </font>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>33</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>33</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>33</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>33</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>33</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>33</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_8">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>33</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_9">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>33</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>230</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 3px solid rgb(164, 0, 0);
background-color: rgb(252, 233, 79);</string>
   </property>
   <property name="text">
    <string>Enter '0' for Blank</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>250</width>
     <height>365</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(173, 127, 168);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_set">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>325</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;SET</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_random">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>295</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Random</string>
   </property>
  </widget>
  <zorder>lbl_back</zorder>
  <zorder>txt_1</zorder>
  <zorder>txt_2</zorder>
  <zorder>txt_3</zorder>
  <zorder>txt_4</zorder>
  <zorder>txt_5</zorder>
  <zorder>txt_6</zorder>
  <zorder>txt_7</zorder>
  <zorder>txt_8</zorder>
  <zorder>txt_9</zorder>
  <zorder>label</zorder>
  <zorder>btn_set</zorder>
  <zorder>Btn_random</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
